sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/LoadflowProvider.java

```diff
@@ -41,11 +42,19 @@
 
     @Override
     public List<SensitivityFactor> getCommonFactors(Network network) {
-        List<SensitivityFactor> factors = new ArrayList<>();
+        return new ArrayList<>();
+    }
 
-        SensitivityVariable defaultSensitivityVariable = defaultSensitivityVariable(network);
+    @Override
+    public List<SensitivityFactor> getAdditionalFactors(Network network) {
+        return  getAdditionalFactors(network, null);
```

Comment:
```null``` is not supposed to be a correct value as second parameter of ```getAdditionalFactors()```. If you want to keep this default value for ```getSensitivityFunctions()``` method, perhaps add a private method layer.